VideoLocker/res/layout/row_discussion_thread.xml

```diff
@@ -4,16 +4,20 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:orientation="horizontal"
     android:paddingBottom="@dimen/edx_margin"
-    android:paddingTop="@dimen/edx_margin"
-    android:gravity="center"
-    android:orientation="horizontal">
+    android:paddingTop="@dimen/edx_margin">
 
     <org.edx.mobile.view.custom.IconImageViewXml
         android:id="@+id/discussion_post_type_icon"
         style="@style/icon_view_standard_size"
-        android:layout_margin="@dimen/edx_margin"
-        app:iconColor="@color/edx_grayscale_neutral_light" />
+        android:layout_marginEnd="@dimen/edx_margin"
+        android:layout_marginLeft="@dimen/edx_margin"
+        android:layout_marginRight="@dimen/edx_margin"
+        android:layout_marginStart="@dimen/edx_margin"
```

Comment:
Note that when you have the same values for left and right, then you don't strictly need to define the relative start and end properties as well.